new/components/nginx/config/default.conf

```diff
@@ -1,5 +1,5 @@
 # Declares the size of a request's body. This is mainly used to allow large blueprints to be uploaded.
-client_max_body_size 50m;
+client_max_body_size 0m;
```

Comment:
Isn't this rather dangerous? If by mistake someone tries to download or upload a 20GB file.. that might create some problems. Maybe we can agree on a logical (albeit arbitrary) limit?